FundParser/FundParser.BL.Tests/Services/FundCsvServiceTests.cs

```diff
@@ -56,32 +56,21 @@ public void SetUp()
     }
 
     [Test]
-    public void UpdateHoldings_DownloadFailed_ThrowsException()
-    {
-        // Arrange
-        downloaderServiceMock.Setup(m => m.DownloadTextFileAsStringAsync(It.IsAny<string>(),
-                It.IsAny<List<(string, string)>>(), It.IsAny<CancellationToken>()))
-            .ReturnsAsync((string)null!);
-
-        // Act & Assert
-        Assert.That(() => fundCsvService.UpdateHoldings(),
-            Throws.Exception.With.Message.EqualTo("Failed to download csv"));
-    }
-
-    [Test]
-    public void UpdateHoldings_ParseFailed_ThrowsException()
+    public async Task UpdateHoldings_ParseFailed_ThrowsException()
```

Comment:
Rename tests name, something like `..._ReturnsNegative`? Not sure. Maybe just `.._ReturnsCorrectCount` will suffice.